model/tps/cics-spans.yaml

```diff
@@ -0,0 +1,28 @@
+groups:
+  - id: span.ibm.cics.http.server
+    type: span
+    stability: development
+    brief: 'This group defines semantic conventions for a HTTP server spans of IBM CICS Transaction Server for z/OS.'
+    extends: span.tps.http.server
+    span_kind: server
+    attributes:
+      - ref: tps.system.name
+        requirement_level: required
+        examples: 'ibm.cics'
+      - ref: ibm.cics.transaction.group.id
+
+  - id: span.ibm.cics.rpc.server
+    type: span
+    stability: development
+    brief: 'This group defines semantic conventions for Remote Procedure Call (RPC) server spans of IBM CICS Transaction Server for z/OS.'
+    extends: span.tps.rpc.server
+    span_kind: server
+    attributes:
+      - ref: tps.system.name
+        requirement_level: required
+        examples: 'ibm.cics'
+      - ref: network.protocol.name
+        examples: ['ipic', 'mro']
```

Comment:
neither of those are defined in the original enum, could you please update the `network.protocol.name` definition?